setup.cfg

```diff
@@ -1,32 +1,75 @@
-[versioneer]
-VCS = git
-style = pep440-branch-based
-versionfile_source = cf_units/_version.py
-versionfile_build = cf_units/_version.py
-tag_prefix = v
-parentdir_prefix = cf_units-
+[metadata]
+author = SciTools Developers
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: MacOS
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX :: Linux
+    Operating System :: Unix
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering
+description = Units of measure as required by the Climate and Forecast (CF) metadata conventions
+download_url = https://github.com/SciTools/cf-units
+keywords =
+    units
+    cf
+    netcdf
+    science
+    oceanography
+    meteorology
+    climate
+license = LGPL-3.0
+license_file = COPYING
+long_description = file: README.md
+long_description_content_type = text/markdown
+name = cf-units
+project_urls =
+    Code = https://github.com/SciTools/cf-units
+    Discussions = https://github.com/SciTools/cf-units/discussions
+    Issues = https://github.com/SciTools/cf-units/issues
+url = https://github.com/SciTools/cf-units
+version = attr: cf_units.__version__
+
+[options]
+include_package_data = True
+install_requires =
+    antlr4-python3-runtime ==4.7.2
+    cftime >=1.2
+    numpy
+    # udunits2 cannot be installed with pip, and it is expected to be
+    #  installed separately.
+packages = find:
+python_requires =
+    >=3.7
+setup_requires =
+    cython
+    numpy
```

Comment:
@trexfeathers We now no longer need this `setup_requires` section, thanks to PEP517/518 support in the `pyproject.toml`

Reply:
https://github.com/SciTools/cf-units/pull/176/commits/81aefb6c793fb403603d523dd8a28d00bbd1984a